pySDC/implementations/sweeper_classes/Runge_Kutta.py

```diff
@@ -58,18 +58,18 @@ def __init__(self, weights, nodes, matrix):
         self.delta_m[0] = self.nodes[0] - self.tleft
 
         # check if the RK scheme is implicit
-        self.implicit = any([matrix[i, i] != 0 for i in range(self.num_nodes)])
+        self.implicit = any([matrix[i, i] != 0 for i in range(self.num_nodes - 1)])
 
 
```

Comment:
We need to make sure people do not specify nodes when using this. See e.g. the Dahlquist example.. you specify the collocation class, but this will be overwritten later. This is confusing.